### PR TITLE
[FIX] Many MAJOR fixes plus dev readme

### LINKED ISSUE #49: Use scale=False and assign a color per number in the experimental design
https://github.com/bigbio/pmultiqc/blob/40c20cf781bf9fed7f113ce4338dc8d86bd3625c/pmultiqc/modules/quantms/quantms.py#L285

Scale = false makes sure that we do not see the weird filled bars in the background. Instead we should use custom bg colors
https://multiqc.info/docs/#custom-cell-background-colours
to color the cell according to the number in the column. (E.g. fraction group 1 is blue, group 2 is red, ...). Use e.g. color brewer Set 2 colors.
One can use the mqc color utils maybe: https://github.com/ewels/MultiQC/blob/master/multiqc/utils/mqc_colour.py

### PR DESCRIPTION
**Pull request by jpfeuffer**
- fixes #49 exp design coloring
- fixes wrong counting of proteins (it counted also protein details)
- major speed and readability fixes
- fixes wrong counting of peptides per protein
- adds short developer readme
- adds gitpod support

### CODE REVIEW

File: pmultiqc/modules/quantms/quantms.py [hunk 1]
```diff
@@ -281,30 +282,35 @@ def draw_exp_design(self):
             'format': '{:,.0f}',  # The header used for the first column
         }
         headers = OrderedDict()
+        set3_scale = mqc_colour_scale(name="Set3")
+        maxnr = len(f_table.index)
+        set3_colors = set3_scale.get_colours(name="Set3")
+        colors = dict( (str(i+1), set3_colors[i % len(set3_colors)]) for i in range(maxnr) )
+        print(colors)
```
Comment (jpfeuffer) [inline pmultiqc/modules/quantms/quantms.py:289]:
```suggestion
```

**Comment by jpfeuffer** [comment]
@ypriverol ping

**Review by ypriverol — APPROVED** [review]

**Comment by ypriverol** [comment]
Incredible work. @WangHong007 some issues assigned to you have been solved by @jpfeuffer .

**Comment by jpfeuffer** [comment]
Only one ;) because I was in contact with the multiQC developer about this.

**Comment by ypriverol** [comment]
Can i have the honors of merging

**Pull request merged**